docs/CHANGELOG.rst

```diff
@@ -1,5 +1,5 @@
-=================
-openfe Change Log
-=================
+=========
+ChangeLog
```

Comment:
```suggestion
Changelog
```

Reply:
Just a small note, I think it is most often Changelog or changelog, but I am happy with ChangeLog as well